lib/src/install/baseline.rs

```diff
@@ -180,7 +180,7 @@ pub(crate) fn install_create_rootfs(
         crate::blockdev::wipefs(dev)?;
     } else if device.has_children() {
         anyhow::bail!(
-            "Detected existing partitions on {}; use e.g. `wipefs` if you intend to overwrite",
+            "Detected existing partitions on {}; use e.g. `wipefs` or --wipe if you intend to overwrite",
```

Comment:
This change looks unrelated...fine by me to sneak in but let's try to split out these types of things into a separate PR so that e.g. if we need to revert a commit due to a bug it doesn't revert unrelated improvements etc.

Reply:
Ah of course sorry

Reply:
#861 861